Dockerfile

```
@@ -1,26 +1,29 @@
 WORKDIR /app
-RUN pip install --upgrade pip
-COPY requirements.txt requirements.txt
-RUN pip3 install --no-cache-dir -r requirements.txt
+
+# Install system dependencies
 RUN apt-get update && \
     apt-get install -y ffmpeg && \
     apt-get clean
+
+# Install Demucs
+RUN python3 -m pip install --upgrade pip
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)